templates/house_view/add_expense.html

```diff
@@ -0,0 +1,24 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %} {% load static %}
+{% load crispy_forms_tags %}
+{% block content %}
+    <link rel="stylesheet" type="text/css" href="{% static 'css/add_expense.css' %}"/>
+    <div id="add_expense">
+        <div class="d-flex justify-content-center h-100">
+            <div class="card">
+                <div class="card-header">
+                    <h3>Add your new expense</h3>
+                </div>
+                <div class="card-body" style="color: rgb(0, 0, 0); background: rgba(252, 252, 252, 0.3); width: 500px;">
+                    <form method="POST">
+                        {% csrf_token %}
+                        <fieldset class="form-group">
+                            {{ form|crispy }}
```

Comment:
We are not using crispy anymore
Can you change to be with material library
you can see in the copy branch I created for you how to do it
@shlior7  cc

Reply:
fixed